src/amp.js

```diff
@@ -0,0 +1,4 @@
+import * as aphrodite from 'aphrodite/no-important';
```

Comment:
My concern is that it will become AMP-specific, since I may need to create a custom `separateStyles` to remove inline `style` attributes.